src/DispatchListener.php

```diff
@@ -211,12 +215,13 @@ protected function marshalBadControllerEvent(
         Application $application,
         \Exception $exception
     ) {
-        $event->setError($application::ERROR_EXCEPTION)
-              ->setController($controllerName)
-              ->setParam('exception', $exception);
+        $event->setName(MvcEvent::EVENT_DISPATCH_ERROR);
+        $event->setError($application::ERROR_EXCEPTION);
```

Comment:
Can't remember why, TBH; that's old code.
On Oct 1, 2015 6:34 AM, "Marco Pivetta" <notifications@github.com> wrote:

> In src/DispatchListener.php
> <https://github.com/zendframework/zend-mvc/pull/31#discussion_r40903533>:
>
> > @@ -211,12 +215,13 @@ protected function marshalBadControllerEvent(
> >          Application $application,
> >          \Exception $exception
> >      ) {
> > -        $event->setError($application::ERROR_EXCEPTION)
> > -              ->setController($controllerName)
> > -              ->setParam('exception', $exception);
> > +        $event->setName(MvcEvent::EVENT_DISPATCH_ERROR);
> > +        $event->setError($application::ERROR_EXCEPTION);
>
> Why a variable here btw?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/zend-mvc/pull/31/files#r40903533>.
>